### DISCUSSION TITLE
Image weird output

### DISCUSSION THREAD

**Discussion by samchouse** [vercel/next.js#23150] [Help] [answered]
When I use an `Image` tag, my image doesn't fit the same as when I use the `img` tag. The reason is that the img that is rendered by next via the `Image` tag has extra styles that override something. How do I fix this?

**Comment by akellbl4** [comment, marked as answer]
Could you please provide some example that reproduces your problem?

**Reply by samchouse** [reply]
If you use `<Image src="/waves.png" layout="fill" objectFill="cover" />` vs `<img src="/waves.svg">` you should see the difference

**Reply by akellbl4** [reply]
When you use `layout="fill"` image has `position: absolute` and you have to wrap it in `<div>` with `position: relative`

**Reply by samchouse** [reply]
![example](https://user-images.githubusercontent.com/46873232/111517053-06103100-872b-11eb-8113-65e4a7665918.png)

```
    <div
      style={{
        position: 'relative',
        height: 320,
        top: 0,
        left: 0,
        width: '100vw'
      }}
    >
      <Image src="/wave.svg" layout="fill" objectFit="cover" quality={100} />
   </div>
```
I still have that white border

Edit: Just set margin 0, got rid of the top border.

**Reply by akellbl4** [reply]
hm, if you'd like to use SVG I see no reason for Image Component. Image Component is more about raster images like png/jpg. I would suggest using SVG as is with `img` for it.

**Reply by akellbl4** [reply]
I've made an example with JPG. It works just fine. https://83y1r.sse.codesandbox.io/
I tried SVG as well, and I noticed that white line. It appears because of the default styles of Image Component.
